### DISCUSSION TITLE
Package doesn't provide -config.cmake file

### DISCUSSION THREAD

**Discussion by appleGun22** [microsoft/vcpkg#35215] [Q&A]
I want to include a package, _miniupnpc_, to my CMakeLists,
but unlike other packges miniupnpc doen't provide a `miniupnpc-config.cmake` file in `vcpkg\installed\triplet\share\`,
how can I include such libraries? until now I've only used `find_package`.

**Comment by dg0yt** [comment]
If the packages don't provide CMake packages, and also no pkg-config .pc files, you can always use traditional `find_path`, `find_library` etc.
